tests/test_url_query.py

```diff
@@ -171,3 +171,23 @@ def test_query_from_empty_update_query(
 
     if "b" in original_url.query:
         assert new_url.query["b"] == original_url.query["b"]
+
+
+@pytest.mark.parametrize(
+    ("original_query_string", "keys_to_drop", "expected_query_string"),
+    [
+        ("a=10&b=20", ["a"], "b=20"),
+        ("a=10&b=20", ["b"], "a=10"),
+        ("a=10&b=20&c=30", ["b"], "a=10&c=30"),
+        ("a=10&b=20&c=30", ["invalid_key"], "a=10&b=20&c=30"),
+        ("a=10&b=20", ["a", "b"], ""),
+        ("a=10&b=20", [], "a=10&b=20"),
```

Comment:
This should probably get some tests for encoded query strings to make sure they don't get double encoded

Reply:
@bdraco I updated test cases.

Reply:
Thanks. Please check for the encoded key case as well. It looks like its only testing values